docs/faq.rst

```diff
@@ -84,6 +84,27 @@ When viewing an individual tool or workflow, you can tell which versions are Ope
 
 :ref:`(back to top) <topFAQ>`
 
+.. _what-is-dockstore-generative-ai-policy:
+
+In human readable terms, what is Dockstore's approach to generative AI?
```

Comment:
IMHO, we can remove "In human readable terms,".

Reply:
:shrug: